.github/workflows/workflow-java.yaml

```diff
@@ -16,10 +17,6 @@ on:
         type: string
         default: "false"
         description: "Build and push a staging tagged container image for this application on commits to the mainline_branch."
-      do_production_build:
-        type: string
-        default: "true"
-        description: "Build and push a production tagged container image for this application on commits to the mainline_branch."
   workflow_dispatch:
```

Comment:
I think `workflow_dispatch` is useless here: It would only apply to triggering on _this_ repo.

Reply:
Oh that's true. Didn't realise that. I'll remove it